app/src/main/res/values/strings.xml

```diff
@@ -18,6 +18,7 @@
   <string name="Device_ID">Device ID</string>
   <string name="debug_push_notifications_register_device">Register device</string>
   <string name="debug_push_notifications_device_registration">Device registration</string>
+  <string name="debug_push_notifications_errored_pldege">Errored pledge</string>
```

Comment:
Just for information, this string is not gonna be translated right?

Reply:
Correct, because it's only used in the Internal Tools!